src/servers/index.files.d.ts

```diff
@@ -1,5 +1,12 @@
+interface Loaders {
+  /* eslint @typescript-eslint/ban-ts-comment: "off" */
+  // @ts-ignore-next-line no-implicit-any
+  txnMetadata
```

Comment:
is there no type for DataLoader in the imported package? also... cant we use same strategy used in trace service?

Reply:
There's the class, but I had a really hard time trying to import it into a `.d.ts` file because of a generics issue (chatGPT was no help either 😅).

We can probably pair to take a look at this actually.

Reply:
We figured this out here https://github.com/GaloyMoney/galoy/pull/2252/commits/1d28952427c72f3beb53a227b06c389d85c209ad